src/components/DefineCompany/AddNameTranslation.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div id="name-translation">
+    <!-- Name Translation form -->
+    <v-form v-model="nameTranslationForm" ref="ntForm" name="name-translation-form">
+      <v-row>
+        <v-col class="pb-0">
+          <v-text-field
+            filled
+            persistent-hint
+            label="Enter Name Translation"
+            v-model="nameTranslation"
+            :rules="nameTranslationRules">
+          </v-text-field>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col class="form__btns pt-0">
+          <v-btn large color="error" id='btn-remove'
```

Comment:
Watch for consistent use of quotes. Typically we use " in the template and ' in the script.

Reply:
oops, good catch thanks! Updated in next commit